docs/MAINTENANCE.md

```diff
@@ -0,0 +1,39 @@
+# Maintenance
+
+This document contains information about project governance and maintenance processes.
+
+
+## Maintainers
+
+The following individuals have generously donated their time to maintaining this project.
+
+* Alex Stewart <alex.stewart@ni.com> - Active. Lead Maintainer.
+* Alex Hearn <alex.hearn@ni.com> - Active. Cooperating Maintainer.
+
+
+## Versioning Policy
+
+This project uses [semantic versioning](https://semver.org/spec/v2.0.0.html), without release candidates. "Release" commits are tagged like `v${major}.${minor}.${patch}`
+
+
+## Release Process
+
+This process should only be performed by a repo maintainer.
+
+1. Create a PGP key associated with your identity.
+2. Register your PGP key for use when signing tags.
+	```bash
+	gpg --list-keys \<$(git config --get user.email)\>  # Copy public key ID from output
+	git config user.signingkey=$pubkey
+	```
+3. Review PRs since the last release. Ensure that all notable changes are represented in the CHANGELOG.
+	1. Commit any CHANGELOG updates.
+4. Tag and sign the HEAD.
+	```bash
+	git tag --sign v${major}.${minor}.${patch}
+	```
+6. Push the tag to github.
+	```bash
+	git push origin v0.0.0
```

Comment:
Should this be `v${major}.${minor}.${patch}`?

Reply:
I used that notation in the prior example to make it very clear that the user should substitute their own values and not just copy the bash scriptlette verbatim. But I didn't do it here, because it is a little more readable and, if the user tried to push tag `v0.0.0`, git would complain that it didn't exist.